crates/ide-completion/src/context/analysis.rs

```diff
@@ -1267,8 +1267,7 @@ fn pattern_context_for(
     pat
         .syntax()
         .ancestors()
-        .skip_while(|it| ast::Pat::can_cast(it.kind()))
-        .next()
```

Comment:
This is wrong

Reply:
Oh I need to get my eyes checked, I thought the negation was missing, nevermind then!